fancyshowcaseview/src/main/java/me/toptas/fancyshowcase/internal/Properties.kt

```diff
@@ -45,7 +46,8 @@ internal data class Properties(
         var dismissListener: DismissListener? = null,
         var queueListener: OnQueueListener? = null,
         var focusedView: IFocusedView? = null,
-        var clickableView: IFocusedView? = null
+        var clickableView: IFocusedView? = null,
+        var typeface: Typeface? = null
```

Comment:
android specific properties are grouped and `AndroidProperties` class. Please move this param under this data class.